android/src/main/AndroidManifest.xml

```diff
@@ -28,6 +28,7 @@
         android:name=".App"
         android:allowBackup="false"
         android:banner="@drawable/tv_banner"
+        android:requestLegacyExternalStorage="true"
```

Comment:
We need this in order to be able to use `Environment.getExternalStoragePublicDirectory(Environment.DIRECTORY_DOWNLOADS)` on Android 10.